### PR TITLE
[8.0][ADD] New module stock_quant_manual_assign

### PR DESCRIPTION
**Pull request by oihane**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please check https://coveralls.io/builds/3892343/source?filename=stock_quant_manual_assign%2Fwizard%2Fassign_manual_quants.py for methods fully not tested. That cases should be treated on tests.

File: stock_quant_manual_assign/README.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+Manual assignment of quants
+===========================
+
+With this module, user can change manually the automatic selection of quants
+
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
```
Comment (pedrobaeza) [inline stock_quant_manual_assign/README.rst:20]: Explain here where you can access the button and how to use it.

File: stock_quant_manual_assign/models/stock_picking.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+from openerp import api, fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = 'stock.picking'
+
+    @api.multi
+    def action_assign(self):
+        self.mapped('pack_operation_ids').unlink()
```
Comment (pedrobaeza) [inline stock_quant_manual_assign/models/stock_picking.py:14]: This one is not needed on odoo/OCB vanilla code, isn't it? Anyway, I think is harmless, but please make sure, and comment why is here.

File: stock_quant_manual_assign/README.rst (hunk 1 above)
Comment (oihane) [inline stock_quant_manual_assign/README.rst:20]: But, this is from the template in https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst
So maybe someone should add what you are telling there :wink:

Comment (pedrobaeza) [inline stock_quant_manual_assign/README.rst:20]: No, I'm telling that for this specific module, explain how to use it, and then add this same thing. We can improve the template adding something like:

```
Usage
=====

<Put here you usage instructions>

.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
   :alt: Try me on Runbot
   :target: https://runbot.odoo-community.org/runbot/153/8.0
```

But it's not what I intend here.

Comment (pedrobaeza) [inline stock_quant_manual_assign/README.rst:20]: The template are contain something like that:

```
Usage
=====

To use this module, you need to:

* go to ...

.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
   :alt: Try me on Runbot
   :target: https://runbot.odoo-community.org/runbot/{repo_id}/{branch}
```

Comment (oihane) [inline stock_quant_manual_assign/README.rst:20]: I thought you were talking about runbot button

File: stock_quant_manual_assign/models/stock_picking.py (hunk 2 above)
Comment (oihane) [inline stock_quant_manual_assign/models/stock_picking.py:14]: To be honest, I do not know why is this code here. I can not explain.

Comment (pedrobaeza) [inline stock_quant_manual_assign/models/stock_picking.py:14]: It's for what I say. Just let it and comment that this is to assure stock.pack.operation are reprocessed.

Comment (oihane) [inline stock_quant_manual_assign/models/stock_picking.py:14]: OK, I added that comment :wink:

**Comment by pedrobaeza** [comment]
:+1:

**Comment by agaldona** [comment]
:+1:

**Comment by Daniel-CA** [comment]
:+1:

**Comment by Sameer1990** [comment]
Hello,
Where i can found this module ?

**Comment by pedrobaeza** [comment]
In https://github.com/OCA/stock-logistics-warehouse/tree/8.0/stock_quant_manual_assign

**Comment by Sameer1990** [comment]
Thanks . :+1:

**Pull request merged**